src/main/java/gameboy/gamepack/racinggame/view/InputView.java

```diff
@@ -1,19 +1,33 @@
 package gameboy.gamepack.racinggame.view;
 
-import java.util.InputMismatchException;
-import java.util.Scanner;
+import java.util.*;
+
+import gameboy.gamepack.racinggame.exception.InvalidRacerNameException;
 
 public class InputView {
 
+    private static final int MAX_NAME_SIZE = 5;
+
     public static int inputNumber(String ask) {
         try {
             display(ask);
             return userIntInput();
-        }catch (InputMismatchException e) {
+        } catch (InputMismatchException e) {
             return inputNumber("숫자만 입력해주세요.");
         }
     }
 
+    public static List<String> inputRacerName(String ask) {
```

Comment:
```suggestion
    public static List<Name> inputRacerName(String ask) {
```
Name을 바로 반환해도 되지 않을까요?
Name 생성자에 유효성 로직이 있다면 보장된 Name을 바로 받을 수 있을 것 같아요